woocommerce/woocommerce
# Single Product Block: Add missing allowed inner blocks to fix nesting placement restrictions

### Changes proposed in this Pull Request:





Closes #45528 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Test scenario 1: Single Product Template

1. Navigate to Single Product template (Appearance > Editor > Templates).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "Single Product" in the block inserter or browse the available blocks until you find them. Add it to your content.
5. Move all blocks from the Columns > Column blocks to be directly nested within the Single Product block. 
6. Make sure you can move all blocks to be directly nested within the Single Product block.
7. Save the template and visit a product page to make sure everything is working as expected.

Test scenario 2: Post

1. Add a new post (Posts > Add new).
2. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
3. Search for "Single Product" in the block inserter or browse the available blocks until you find them. Add it to your content.
4. Move all blocks from the Columns > Column blocks to be directly nested within the Single Product block.
5. Make sure you can move all blocks to be directly nested within the Single Product block.
6. Save the post and visit it to make sure everything is working as expected.

Test scenario 3: Page

1. Add a new post (Page > Add new).
2. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
3. Search for "Single Product" in the block inserter or browse the available blocks until you find them. Add it to your content.
4. Move all blocks from the Columns > Column blocks to be directly nested within the Single Product block.
5. Make sure you can move all blocks to be directly nested within the Single Product block.
6. Save the page and visit it to make sure everything is working as expected.

#### Before

https://github.com/woocommerce/woocommerce/assets/20469356/10ec6931-c7c9-4fe8-b4c8-edf21361a55a

#### After

https://github.com/woocommerce/woocommerce/assets/20469356/3fb17d00-b01a-444a-b66d-58768ab4c8ab



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Fix issue preventing some blocks from being direct children of the Single Product block.

#### Comment 

</details>
